src/Core/Services/Implementations/AmazonSesMailDeliveryService.cs

```diff
@@ -21,6 +22,7 @@ public class AmazonSesMailDeliveryService : IMailDeliveryService, IDisposable
         private readonly string _source;
         private readonly string _senderTag;
         private readonly string _configSetName;
+        private const int MAX_BATCH_MESSAGE_RECIPIENTS = 50;
```

Comment:
Did we change how we define const? Previously they have all been PascalCased. These guidelines show they should be at the top as well: https://github.com/bitwarden/company/wiki/Code-Style-Guidelines

Reply:
no, just old habits. Thanks